### ISSUE TITLE
bug: verify does not work with Elements

### ISSUE THREAD

**Issue by throwException** [8go/matrix-commander-rs#110]
When I start `matrix-commander-rs --verify` and then click verification in the Elements app, nothing happens.

Might this be the result of Elements using in-room-verification instead of device verification?

**Comment by 8go** [comment]
First of all, before emoji verification, I did manual verification `matrix-commander-rs  --bootstrap --verify manual -d`.

Then try emoji verification again, I could verify between 2 accounts. One account on Element on cell phone who initiates the verification. Second account on the `matrix-commander-rs` CLI running the `--verify` option. I started the `matrix-commander-rs --verify` first to sit waiting, then the cell phone did the "request verification". 

I could also do it between the `matrix-commander-rs` CLI and Element on a web browser. Here I tested two different devices of the same user.

In short, I could get both Element website as well as Element app on cell phone working for me.

Try again with v0.4.0.

**Comment by 8go** [comment]
In version v0.5.0 I added outgoing requests, i.e. `matrix-commander-rs` send a request (on invitation) to verify to another client (e.g. Element). This seems to work on Element webpage in browser. But this definitely does not work for me on Element app on Android. 

Matrix has changed the spec for verification and also the implementation of verification in Element. So, some things that used to work no longer work. Also some things work in Element webpage, but not in Element phone app. Things are different in different implementations. 

Not sure if there is a bug in Element phone app or in `matrix-commander-rs`.  `matrix-commander-rs` does not receive any event from phone after user accepts the verification in the verification popup.

**Comment by fidoman** [comment]
"Elements using in-room-verification"
When verifying with other user it is true, DM room must be created. Just send message to matrix-commander, read it reply, then emoji with other user proceeds.
When verifying with other own device it is not needed.

**Comment by Benjamin-Loison** [comment]
<details>
<summary>Personal notes:</summary>

For reference on my Linux Mint 22.3 Cinnamon Framework 13 Virtual Machine Manager LineageOS 23.0 virtual machine:

```bash
matrix-commander --verify emoji
```
<details>
<summary>Output:</summary>

```
Ready and waiting ...
Go to other Matrix client like Element and initiate Emoji verification there.
Best to have the other Matrix client ready and waiting before you start
matrix-commander-rs.
Accepting verification request from @overclock3000:matrix.org
Starting verification with @overclock3000:matrix.org pPzimJaVcr
```
</details>

<img width="1280" height="772" alt="Image" src="https://github.com/user-attachments/assets/f34df473-9d00-4c2a-9601-013a68da5ab6" />

```bash
matrix-commander --verify emoji-req --device pPzimJaVcr
```
<details>
<summary>Output:</summary>

```
Ready and waiting ...
We send request to other Matrix client like Element and initiate Emoji 
verification with them. Best to have the other Matrix client ready and 
waiting before you start matrix-commander-rs.

 ### THIS IS PARTIALLY BROKEN. DOES NOT SEEM TO WORK WITH ELEMENT ANDROID APP. ###
 ### BUT IT DOES WORK WITH ELEMENT WEB APP IN BROWSER. ###

Sending request to user's "@overclock3000:matrix.org" device "pPzimJaVcr".
Starting verification with @overclock3000:matrix.org pPzimJaVcr
```
</details>

```bash
matrix-commander --devices
```
<details>
<summary>Output:</summary>

```
Device: 80w4HjKhrw Element desktop Linux Mint 22.2 Cinnamon Framework 13
Device: ELLn6nBKko instance-20230827-2235_matrix-commander-rs
Device: O8vGswb6qf OverClock3000 matrix-commander-rs
Device: R5m2vXMFK9 matrix-commander-rs_pegasus_debian
Device: bzUymzfjAI LemnosLife VPS matrix-commander-rs
Device: pPzimJaVcr
```
</details>

<img width="1280" height="772" alt="Image" src="https://github.com/user-attachments/assets/d1e05088-99c7-4ab7-9b37-cd5d3dec7f5f" />

<img width="1280" height="772" alt="Image" src="https://github.com/user-attachments/assets/1ea45644-5c67-42c8-87ae-4ce55df4c2e4" />

<img width="1280" height="772" alt="Image" src="https://github.com/user-attachments/assets/d07ff4c5-bd4d-40d5-9169-afb406d08ae8" />

<img width="1280" height="772" alt="Image" src="https://github.com/user-attachments/assets/e92f3c93-cc60-49d7-8fd4-bcf8b7f76323" />

<img width="1280" height="772" alt="Image" src="https://github.com/user-attachments/assets/d1ffa9f9-811e-4697-8f1c-e1cf3b6a612a" />
</details>

So both `--verify emoji{,-req}` methods have the same ever loading issue, while I do not have any issue on Element desktop.

**Comment by Benjamin-Loison** [comment]
However, with Element desktop initiative I also have at least for tens of seconds loading emojis to verify Element Android (I have not waited more)... From Element Android initiative, emojis load fine on Element desktop and I am able to verify them.
